src/main/java/codeu/model/data/User.java

```diff
@@ -21,29 +21,29 @@
 public class User {
   private final UUID id;
   private final String name;
-  private final String password;
+  private final String hashedPassword;
   private final Instant creation;
 
   /**
    * Constructs a new User.
    *
    * @param id the ID of this User
    * @param name the username of this User
-   * @param password the password of this User
+   * @param hashedPassword the hashedPassword of this User
    * @param creation the creation time of this User
    */
   public User(UUID id, String name, String password, Instant creation) {
     this.id = id;
     this.name = name;
-    this.password = password;
+    this.hashedPassword  = password;
     this.creation = creation;
   }
   
   /**
   * Returns the password of this User.
   */
   public String getPassword(){
```

Comment:
Since you're renaming the field to hashedPassword, I think you should also rename this method to getHashedPassword().